extension/css/relay.css

```diff
@@ -19,49 +19,43 @@
   src: url("/fonts/Metropolis/Metropolis-Light.woff2") format("woff2");
 }
 
-.first-run,
-.new-menu-wrapper,
-.panel,
-.relay-icon,
-.relay-modal-wrapper {
-  --blue2: #0090ed;
-  --blue3: #0060df;
-  --blue4: #0250bb;
-  --blueHover: #003eaa;
-  --blueActive: #002275;
-  --blueButtonFocus: 0 0 1px 1px #0a84ff, 0 0 0 4px rgba(10, 132, 255, 0.3);
-  --fxGradient: linear-gradient(-90deg, #ff9100 0%, #f10366 50%, #6173ff 100%);
-  --grey10: hsl(240, 9%, 98%);
-  --grey20: #ededf0;
-  --grey30: #d7d7db;
-  --grey40: #b1b1b3;
-  --grey50: #737373;
-  --grey60: #4a4a4f;
-  --grey70: #38383d;
-  --grey80: #2a2a2e;
-  --grey90: #0c0c0d;
-  --ink: #20123a;
-  --ink70: #363959;
-  --InterUI: "Inter UI", -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  --Metropolis: "Metropolis", sans-serif, "SF Pro Text", "Segoe UI", Roboto, "Open Sans";
-  --panelWidth: 360px;
-  --panelHeight: 400px;
+.fx-relay-first-run,
+.fx-relay-menu-wrapper,
+.fx-relay-panel,
+.fx-relay-icon,
+.fx-relay-modal-wrapper {
+  --relayBlue3: #0060df;
+  --relayBlue4: #0250bb;
```

Comment:
Random Q: Why are these `--relayBlue3` and `--relayBlue4` and not `--relayBlue30` to match the `--relayGrey30` style below?